microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/security/ListenerSecurityConfigurationTest.java

```diff
@@ -111,6 +112,8 @@ public void setup() throws Exception {
                 //  MP OpenAPI up
                 OpenApiServerConfiguration.enableOpenApi(client);
             }
+
+            String jbossHostName = arquillianContainerProperties.getDefaultManagementAddress();
```

Comment:
I know that such situation is very unlikely to happen but management address and hostname (for deployments) might differ. 

Would it be possible to extract this from URL for given deployment?

Reply:
Hi @mnovak1 this should be fixed now.